multinet/multinet/__init__.py

```diff
@@ -47,7 +51,24 @@ def validate_csv(rows):
                 uniqueKeys.add(key)
 
         if (len(duplicates) > 0):
-            raise RestException(f'CSV Validation Failed: Duplicate Keys {", ".join(duplicates)}.')
+            raise RestException(f'{BASE_ERROR_MSG} Duplicate Keys {", ".join(duplicates)}.')
+    elif '_from' in rows.fieldnames and '_to' in rows.fieldnames:
+        # Edge Table, check that each cell has the correct format
+        validCell = re.compile('[^/]+/[^/]+')
```

Comment:
Could you change `validCell` to `valid_cell`?

Reply:
Sure thing